ote/src/cljc/ote/util/collections.cljc

```diff
@@ -0,0 +1,7 @@
+(ns ote.util.collections)
+
+(defn remove-by-index
+  "Remove element from vector by index."
+  [vector idx]
+  (into (subvec vector 0 idx)
+        (subvec vector (inc idx))))
```

Comment:
I'm kind of sure that this won't work when removing last element

Reply:
It does.

Reply:
I tested this. It works.